### ISSUE TITLE
variant tags task too long

### ISSUE THREAD

**Issue by davmlaw** [SACGF/variantgrid#1153]
This takes too long and dies

We should split this up somehow so it runs in parallel?

[See rollbar](https://app.rollbar.com/a/jimmy.andrews/fix/item/VariantGrid/5349)

**Comment by davmlaw** [comment]
I split out clingen into per-VCF - this worked well but still spending a very long time in variant tags task

Loading variants - 187,786 is taking 450+ seconds til I killed it.

We only need variant ID so I'll see if loading just that saves a bit of time

**Comment by davmlaw** [comment]
Re-importing now: https://variantgrid.com/upload/view_upload_pipeline/7321

**Comment by davmlaw** [comment]
Completed successfully
